<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DP::VPN GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="MainTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="allServersTab">
          <attribute name="title">
           <string>Servers</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>251</width>
             <height>311</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="allServersLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QPushButton" name="buttonAddServer_2">
                <property name="text">
                 <string>Add Server</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="AllServers"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="ServerLogTab">
          <attribute name="title">
           <string>Logs</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>271</width>
             <height>291</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="serverLogLayout">
            <item>
             <widget class="QTextEdit" name="ServerLog">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="proxiesTab">
          <attribute name="title">
           <string>Proxies</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_6">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>491</width>
             <height>271</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="proxiesTabLayout">
            <item>
             <widget class="QListView" name="proxiesView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="proxy_name"/>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Type:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="proxy_proxyType"/>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Server:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="proxy_server"/>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="proxy_port"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QPushButton" name="proxy_delete">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="proxy_add">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="proxy_error">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="statusTab">
          <attribute name="title">
           <string>Status</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>551</width>
             <height>301</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="statusTabLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>IP:</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Local:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelLocalIP">
                    <property name="text">
                     <string>NONE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Global:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelGlobalIP">
                    <property name="text">
                     <string>NONE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Traffic</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Sended:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelSendedTraffic">
                    <property name="text">
                     <string>NONE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Receaved:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelReceavedTraffic">
                    <property name="text">
                     <string>NONE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonUpdateStat">
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="settingsTab">
          <attribute name="title">
           <string>Settings</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>571</width>
             <height>301</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="settingsTabLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Proxy:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="settingProxy"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Protocol:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="settings_protocol"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="StatusLabel">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="Status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="buttonDeleteServer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonConnect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
